bip-musig2.mediawiki

```diff
@@ -162,10 +166,10 @@ The purpose of specifying tweaking is to ensure compatibility with existing uses
 The MuSig2 algorithms take arbitrary tweaks as input but accepting arbitrary tweaks may negatively affect the protocol's security.<ref>It is an open question whether allowing arbitrary tweaks from an adversary affects the unforgeability of MuSig2.</ref>
 Instead, signers should obtain the tweaks according to other specifications.
 This typically involves deriving the tweaks from a hash of the aggregate public key and some other information.
-Depending on the specific scheme that is used for tweaking, the aggregate public key is required in different formats.
-In addition to ''GetXonlyPubkey'', this specification defines the ''GetPlainPubkey'' algorithm which returns the aggregate public key in ''compressed'' format.
-This format is 33 bytes in size and is used, for example, in [https://github.com/bitcoin/bips/blob/master/bip-0032.mediawiki BIP32].
+Depending on the specific scheme that is used for tweaking, either the plain or the X-only aggregate public key is required.
+For example, [https://github.com/bitcoin/bips/blob/master/bip-0032.mediawiki BIP32] uses the former as input to the hash that determines the tweak, and [https://github.com/bitcoin/bips/blob/master/bip-0341.mediawiki BIP341] uses the latter.
```

Comment:
Another possible wording:

```suggestion
For example, [https://github.com/bitcoin/bips/blob/master/bip-0032.mediawiki BIP32] applies the tweak to the plain key, whereas [https://github.com/bitcoin/bips/blob/master/bip-0341.mediawiki BIP341] converts the key to X-only before determining (e.g. with a hash function) and applying the tweak.
```

Reply:
Hm I tried to say that you need the keys to compute the tweak, not to apply it. I rephrased the sentence.

Reply:
The emphasis on how to compute the tweak makes sense to me, thanks for clarifying.